blueprints.yaml

```diff
@@ -321,4 +321,23 @@ form:
       label: Visitors history
       default: 20
 
+    Editor:
+      type: section
+      title: Editor
```

Comment:
`Frontmatter` would probably be more appropriate than `Editor`. Editor is very generic and doesn't really refer to the frontmatter only. The Content, for instance, is using the editor field, which would make this set of options a bit confusing.